lib/phoenix_live_view/tag_engine.ex

```diff
@@ -140,6 +140,10 @@ defmodule Phoenix.LiveView.TagEngine do
   for more information.
   """
   defmacro inner_block(name, do: do_block) do
+    # TODO: It is unfortunate this macro is defined in TagEngine
+    # but Engine has intrinstic knowledge about it. Perhaps it
+    # should be moved to the engine instead with the catch-all
+    # clause removed (it exists only for backwards compat).
```

Comment:
We cannot move it completely as it is part of the public TagEngine API, we could only deprecate it.